packages/blade/src/components/Typography/Heading/Heading.tsx

```diff
@@ -16,11 +17,16 @@ type HeadingCommonProps = {
 
 type HeadingNormalVariant = HeadingCommonProps & {
   variant?: Exclude<HeadingVariant, 'subheading'>;
+  size?: HeadingSize;
   weight?: keyof Theme['typography']['fonts']['weight'];
 };
 
 type HeadingSubHeadingVariant = HeadingCommonProps & {
   variant?: Extract<HeadingVariant, 'subheading'>;
+  /**
+   * `size` cannot be used with variant="subheading". Either change to variant="regular" or remove size prop
+   */
```

Comment:
Suggestion: add a default for JSDoc

Reply:
Done. Added it to the other `HeadingNormalVariant` type since the actual type was defined there. We're just overriding here to undefined to make sure people don't pass size with `variant="subheading"`